_includes/release-notes/51new-features.md

```diff
@@ -0,0 +1,229 @@
+
+The following new features and enhancements were added for the {{site.productname}} 5.1 release.
+
+### Mobile
+
+{{site.productname}} 5.1 provides an improved mobile editor, replacing the existing mobile editor with a touch friendly version of the silver theme.
+
+The new mobile experience comes with a number of user interface changes, building on the {{site.productname}} desktop experience:
+
+* [Mobile devices now use the `silver` theme](#mobiledevicesnowusethesilvertheme).
+* [Horizontal contextual menus on mobile](#horizontalcontextualmenusonmobile).
+* [Side-scrolling toolbars on mobile](#side-scrollingtoolbarsonmobile).
+* [Contextual keyboard settings](#contextualkeyboardsettings).
+* [New mobile defaults for selected settings](#newmobiledefaultsforselectedsettings).
+* [Table cell selection on mobile](#tablecellselectiononmobile).
+* [Touch-friendly split buttons](#touch-friendlysplitbuttons).
+
+The new mobile experience allows most of the {{site.productname}} plugins to work on mobile devices, except for:
+
+* [Comments]({{site.baseurl}}/plugins/comments/).
+* [Image tools]({{site.baseurl}}/plugins/imagetools/).
+* [MoxieManager]({{site.baseurl}}/plugins/moxiemanager/).
+* [Paste]({{site.baseurl}}/plugins/paste/).
+* [Permanent Pen]({{site.baseurl}}/plugins/permanentpen/).
+* [Print]({{site.baseurl}}/plugins/print/).
+
+To ensure the mobile editor functions as intended, add the following `meta` tag to the `head` of pages using {{site.productname}}.
```

Comment:
Do we have any way of highlighting this so it's super obvious? People missing this would be a bit annoying...

Reply:
Added the heading 
> #### Enabling the mobile editor